specification/2.0/schema/material.schema.json

```diff
@@ -5,28 +5,79 @@
     "description" : "The material appearance of a primitive.",
     "allOf" : [ { "$ref" : "glTFChildOfRootProperty.schema.json" } ],
     "properties": {
-        "technique" : {
-            "allOf" : [ { "$ref" : "glTFid.schema.json" } ],
-            "description" : "The ID of the technique.",
-            "gltf_detailedDescription" : "The ID of the technique.  If this is not supplied, and no extension is present that defines material properties, then the primitive should be rendered using a default material with 50% gray emissive color."
-        },
-        "values" : {
-            "type" : "object",
-            "description" : "A dictionary object of parameter values.",
-            "properties" : {
-            },
-            "additionalProperties" : {
-                "$ref" : "material.values.schema.json"
-            },
-            "default" : {},
-            "gltf_detailedDescription" : "A dictionary object of parameter values.  Parameters with the same name as the technique's parameter override the technique's parameter value."
+        "baseColorFactor" : {
```

Comment:
I assume that the valid ranges are still being defined? (Just wondering, because in the README, the ranges for `metallicFactor` and `roughnessFactor` are already given, but here, only  the `roughnessFactor` has a `minimum` and `maximum`...)